packages/text-bitmap/src/formats/XMLStringFormat.ts

```diff
@@ -19,9 +19,14 @@ export class XMLStringFormat
      */
     static test(data: unknown): boolean
     {
-        const xmlStringRegExp = new RegExp(/<font>(\s+)?<info\s+face=/);
+        if (typeof data === 'string' && data.includes('<font>'))
```

Comment:
Could you use `indexOf` instead of `includes`, since `includes` [is not ES5](https://caniuse.com/es6-string-includes) and not supported on IE11.